test/integrationTests/signatureHelp.integration.test.ts

```diff
@@ -87,6 +87,12 @@ suite(`SignatureHelp: ${testAssetWorkspace.description}`, function () {
         expect(c.signatures[0].parameters[c.activeParameter].label).to.equal(answer);
     });
 
+    test("Signature Help identifies active parameter based on comma for multpilte commas", async function () {
+        let c = <vscode.SignatureHelp>await vscode.commands.executeCommand("vscode.executeSignatureHelpProvider", fileUri, new vscode.Position(18, 25));
```

Comment:
Bonus points: replace the many instances of ` = <vscode.SignatureHelp>await vscode.commands.executeCommand("vscode.executeSignatureHelpProvider", fileUri, new vscode.Position(` with calls to a function that takes a fileuri and position and returns a vscode.SignatureHelp.